drivers/virtualbox/virtualbox.go

```diff
@@ -533,7 +576,7 @@ func (d *Driver) diskPath() string {
 func (d *Driver) generateDiskImage(size int) error {
 	log.Debugf("Creating %d MB hard disk image...", size)
 
-	magicString := "boot2docker, please format-me"
+	magicString := "data"
 
 	buf := new(bytes.Buffer)
 	tw := tar.NewWriter(buf)
```

Comment:
Yeah I think so.  I'm not too familiar with the disk creation.  I will give it a shot.  Thanks!